tests/guestlog/guestlog.go

```diff
@@ -150,6 +152,55 @@ var _ = Describe("[sig-compute]Guest console log", decorators.SigCompute, func()
 				Expect(outputString).To(ContainSubstring("\r\n" + testString + "\r\n"))
 			})
 
+			It("it should fetch logs for a running VM with logs API", func() {
```

Comment:
"Let's keep a single test based on `kubectl logs`to avoid decreasing the functional coverage."

What do we decrease? Using the Go API is what the kubectl is using. I think we don't need this.

Reply:
Yes, it sounds  reasonable.
Let's keep it simpler and test just with the API assuming that the CLI client is properly doing its job.